src/corecel/io/OutputRegistry.cc

```diff
@@ -41,7 +41,7 @@ void OutputRegistry::insert(SPConstInterface interface)
 
     Category cat = interface->category();
     auto [prev, inserted]
-        = interfaces_[cat].insert({std::move(label), std::move(interface)});
+        = interfaces_[cat].insert({std::string{label}, std::move(interface)});
```

Comment:
we can also use a `string_view` as a key here

Reply:
Is the lifetime of `label` guaranteed to be longer than the lifetime of the`OutputRegistry`? (the `std::move` in the existing code suggest otherwise).

Reply:
Actually that's a really good point. If you store a string view from an rvalue, you might be exceeding the lifetime of the object.